apps/query-ui/queryui/Sycamore_Query.py

```diff
@@ -141,11 +150,13 @@ def main():
         show_schema(client, st.session_state.index)
         with st.form("query_form"):
             st.text_input("Query", key="query")
-            col1, col2 = st.columns(2)
+            col1, col2, col3 = st.columns(3)
             with col1:
                 submitted = st.form_submit_button("Run query")
             with col2:
                 st.toggle("Plan only", key="plan_only", value=False)
+            with col3:
+                st.toggle("Local mode", key="local_mode", value=False)
```

Comment:
I think "local mode" is a misnomer here since technically everything is local, so maybe change this to "Use Ray" and invert the meaning of the flag.

Reply:
the sycamore exec mode is local so it makes sense I think? happy to change if you feel strongly

Reply:
Cool I'll update the UI visuals to say "Use Ray" but let the internal/api continue to use sycamore terminology